sec_certs_page/templates/common/base.html.jinja2

```diff
@@ -300,76 +362,77 @@
+                                    <a href="https://muni.cz" target="_blank" rel="noopener">
+                                        <img id="muni-logo" src="{{ url_for("static", filename="img/muni_light.svg") }}"
+                                             alt="Masaryk University" style="max-width: 70%" loading="lazy"/>
+                                    </a>
+                                </div>
+                            
+                                <div class="flex-grow-1 flex-shrink-1 flex-basis-0">
+                                    <h5 class="mb-4">Project</h5>
+                                    <p><a href="https://github.com/crocs-muni/sec-certs" target="_blank" class="link-light link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">Github</a></p>
+                                    <p><a href="{{ url_for('about.index') }}" target="_blank" class="link-light link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">About</a></p>
+                                    <p><a href="{{ url_for('about.research') }}" target="_blank" class="link-light link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">Research</a></p>
+                                </div>
+                                <div class="flex-grow-1 flex-shrink-1 flex-basis-0">
+                                    <h5 class="mb-4">Quick links</h5>
+                                    <p><a href="{{ url_for("docs.serve_docs", path="index.html") }}"
+                                            class="link-light link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">
+                                                Documentation
+                                        </a>
+                                    </p>
+                                    <p><a href="{{ url_for('about.privacyPolicy') }}" class="link-light link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">Privacy policy</a></p>
+                                    {% if current_user.is_authenticated %}
+                                        <p><a href="{{ url_for('admin.logout') }}" class="link-danger link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">Logout</a></p>
+                                    {% else %}
+                                        <p><a href="{{ url_for('admin.login') }}" class="link-light link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">Admin login</a></p>
+                                    {% endif %}
+                                </div>
+                            
+                                <div class="flex-grow-1 flex-shrink-1 flex-basis-0">
+                                    <h5 class="mb-4">Contact us</h5>
+                                    <p><a href="#" class="link-light link-offset-2 link-underline-opacity-25 link-underline-opacity-100-hover">hello@sec-certs.org</a></p>
```

Comment:
We don't really have a hello@ email. But it all goes to the same address so this is fine for the time being.